libs/components/core/src/lib/modules/affix/affixer.ts

```diff
@@ -190,6 +191,15 @@ export class SkyAffixer {
   #affix(): void {
     const offset = this.#getOffset();
 
+    // In Safari on iOS, fixed position elements may scroll with the viewport.
+    if (offset && this.#config.position !== 'absolute') {
+      const { top, left } = getViewportOffset();
+      offset.top = (offset.top || 0) + top;
+      offset.left = (offset.left || 0) + left;
+      offset.bottom = (offset.bottom || 0) + top;
+      offset.right = (offset.right || 0) + left;
+    }
+
```

Comment:
This is 1/2 of the fix for mobile Safari, which was the original issue where we tried using absolute position.